iHome/ihome.frontend/src/app/models/bill.ts

```diff
@@ -0,0 +1,8 @@
+export interface Bill {
```

Comment:
Bill? Co to takiego?

Reply:
A no bo to ma być nowa funkcjonalność która miała być w innym PRze